app/controllers/api/v2/apps_controller.rb

```diff
@@ -136,6 +138,52 @@ def update_barito_app
    render json: app_response
   end
 
+  def update_barito_app_labels
+    valid, error_response = validate_required_keys(
+      [:app_group_secret, :app_name])
+    render json: error_response, status: error_response[:code] and return unless valid
+
+    app_group = AppGroup.find_by(secret_key: params[:app_group_secret])
+    if app_group.blank? || !app_group.available?
+      render json: {
+        success: false,
+        errors: ['AppGroup not found or inactive'],
+        code: 404
+      }, status: :not_found and return
+    end
+    
+    app = BaritoApp.find_by(
+      app_group: app_group,
+      name: params[:app_name]
+    )
+    if app.blank?
+      render json: {
+        success: false,
+        errors: ['App not found or inactive'],
+        code: 404
+      }, status: :not_found and return 
+    elsif !app.available?
+      render json: {
+        success: false,
+        errors: ["App is inactive"],
+        code: 503
+      }, status: :service_unavailable and return
+    else
+      app.update(
+        labels: params[:labels],
```

Comment:
do we also need to add this label keys as `validate_required_keys`

Reply:
no need mas @fadlinurhasan , since we can make the labels empty `{}`, we can't make labels as require keys, I'll go make a default value as `{}` if the `params[:labels]` is blank

Reply:
https://github.com/BaritoLog/BaritoMarket/pull/270/commits/334d315fb1d1b67ac06cd376286adabae1af7359